.gitignore

```diff
@@ -3,3 +3,5 @@ Gemfile.lock
 .bundle
 vendor
 .idea
+lib/com
+lib/logstash-filter-geoip_jars.rb
```

Comment:
I don't know, this is a auto-generated file that will be overriden by jar-dependencies rake task after downloaded jars from Maven.